Redesign: grid view experiment

bwindels opened the pull request.

dbkr commented on res/css/structures/_GroupGridView.scss:
```diff
@@ -0,0 +1,131 @@
+
+.mx_GroupGridView_activeTile:before {
+    border-radius: 14px;
+    border: 8px solid rgba(134, 193, 165, 0.5);
```
should these be colour constants in the theme?

ara4n replied:
(yes please!)

dbkr commented on res/css/structures/_GroupGridView.scss:
```diff
@@ -0,0 +1,131 @@
+    display: block;
+    margin-top: 100px;
+    text-align: center;
+    content: "no room in this tile yet";
```
could this be done in a way that can be i18ned?

dbkr commented on src/components/structures/GroupGridView.js:
```diff
@@ -0,0 +1,135 @@
+            default:
+                break;
+        }
+    }
```
Legacy from a different time?

dbkr commented on src/components/views/rooms/MessageComposerInput.js:
```diff
@@ -344,12 +354,19 @@ export default class MessageComposerInput extends React.Component {
+        if (this.props.isGrid) {
+
+
+
```
spacey!

dbkr commented on src/stores/RoomViewStore.js:
```diff
@@ -335,8 +306,7 @@ class RoomViewStore extends Store {
+const MatrixDispatcher = require("../matrix-dispatcher");
+const blubber = new RoomViewStore(new MatrixDispatcher());
+
+export default blubber;
```
Think I preferred the old name ;)

bwindels commented:
Thanks for the review @dbkr, could have polished it a bit more indeed. I addressed your comments, but while testing got a missing roomViewStore prop error on the RoomView, so marking it again as notready until I figure out why...
